src/main/java/org/craftercms/studio/permissions/StudioPermissionsConstants.java

```diff
@@ -45,6 +45,7 @@ public final class StudioPermissionsConstants {
     public static final String PERMISSION_LIST_REMOTES = "list_remotes";
     public static final String PERMISSION_PUBLISH = "publish";
     public static final String PERMISSION_PUBLISH_STATUS = "publish_status";
+    public static final String PERMISSION_PUBLISH_BY_COMMITS = "publish_by_commits";
```

Comment:
Please note in the api spec that this permission is now required

Reply:
I updated this.